server/server.go

```diff
@@ -1603,6 +1629,17 @@ func (s *Server) Start() {
 		})
 	}
 
+	// Setup OCSP Stapling monitors.
+	// This will abort server from starting if there are no valid staples
+	// and OCSP policy is Always or Must Staple.
+	if oc := opts.OCSPConfig; oc != nil {
```

Comment:
If `oc == nil` or if it's not nil and the mode is Auto, and the certificate says MustStaple, then we need to abort for that scenario too.

Reply:
If cert has MustStaple flag then it activates Must staple mode.